api/integrations/github/github.py

```diff
@@ -16,14 +18,88 @@
     LINK_SEGMENT_TITLE,
     UNLINKED_FEATURE_TEXT,
     UPDATED_FEATURE_TEXT,
+    GitHubTag,
 )
 from integrations.github.dataclasses import GithubData
 from integrations.github.models import GithubConfiguration
 from integrations.github.tasks import call_github_app_webhook_for_feature_state
+from projects.tags.models import Tag, TagType
 from webhooks.webhooks import WebhookEventType
 
 logger = logging.getLogger(__name__)
 
+tag_by_event_type = {
+    "pull_request": {
+        "closed": GitHubTag.PR_CLOSED.value,
+        "converted_to_draft": GitHubTag.PR_DRAFT.value,
+        "opened": GitHubTag.PR_OPEN.value,
+        "reopened": GitHubTag.PR_OPEN.value,
+        "ready_for_review": GitHubTag.PR_OPEN.value,
+        "merged": GitHubTag.PR_MERGED.value,
+    },
+    "issues": {
+        "closed": GitHubTag.ISSUE_CLOSED.value,
+        "opened": GitHubTag.ISSUE_OPEN.value,
+        "reopened": GitHubTag.ISSUE_OPEN.value,
+    },
+}
```

Comment:
This section should probably live in the `constants.py` module.

Reply:
I was tempted to put it in constants, however, it would lose context and require extra comments since it's used only in the context of the github.py file. i.e. the event_type in its name refers to a webhook request event type.